### ISSUE TITLE
NPM Official Latest Version of ReactXP is 0.42. Should it be updated?

### ISSUE THREAD

**Issue by fbartho** [Microsoft/reactxp#348]
See: https://www.npmjs.com/package/reactxp

```bash
$ yarn outdated
yarn outdated v1.2.1
Package Current Wanted Latest  Package Type URL                                 
reactxp 0.46.4  0.46.5 0.42.12 dependencies https://microsoft.github.io/reactxp/
Done in 0.78s.
```

**Comment by erictraut** [comment]
We continue to publish bug fixes on the 0.42.x branch in parallel with 0.46.x versions from master. The npm site will show whatever version happened to be published last. You can see the full list of published versions if you type "npm view reactxp" from a command line.

**Comment by fbartho** [comment]
Thanks!
